### PR TITLE
Update GeoJsonWriteDriver to write only rfc7946 GeoJson files

### PR DESCRIPTION
**Pull request by gpbp**
I used also ST_Transform to write all coordinates into rfc 7946 coordinates #820

### CODE REVIEW

**Comment by ebocher** [comment]
Please refer this PR to an issue and resolve conflicts

File: h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonWriteDriver.java [hunk 1]
```diff
@@ -376,10 +376,12 @@ private void write(GeometryCollection coll, JsonGenerator gen) throws IOExceptio
      * @param gen
      * @throws IOException
      */
-    private void write(Polygon geom, JsonGenerator gen) throws IOException {
+    private void write(Polygon geom, JsonGenerator gen) throws IOException, SQLException {
         gen.writeStringField("type", "Polygon");
         gen.writeFieldName("coordinates");
         gen.writeStartArray();
+        ST_Transform transform = new ST_Transform;
+        geom = (Polygon) transform.ST_Transform(connection, geom, 4326);
```
Comment (ebocher) [inline h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonWriteDriver.java]: I'm not in favour of a geometry transformation. The geometry SRID must be managed by the user. It's up to the user to send the good CRS.

File: h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonWriteDriver.java [hunk 2]
```diff
@@ -499,36 +501,4 @@ public boolean isSupportedPropertyType(int sqlTypeId, String sqlTypeName) throws
...
     }
-
-    /**
-     * Write the CRS in the geojson
-     *
-     * @param jsonGenerator
-     * @param authorityAndSRID
-     * @throws IOException
-     */
-    private void writeCRS(JsonGenerator jsonGenerator, String[] authorityAndSRID) throws IOException {
```
Comment (ebocher) [inline h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonWriteDriver.java]: Do not remove this method because we can use it later.

Comment (gpbp) [inline h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonWriteDriver.java]: ok.

File: h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonWriteDriver.java (hunk 1 above)
Comment (gpbp) [inline h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonWriteDriver.java]: Should we stock coordinates in wgs84 while parsing or convert them into wgs84 while writing

Comment (gpbp) [inline h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonWriteDriver.java]: in rfc7946 we only can use wgs 84. So when we write GeoJson file, coordinates should be converted into wgs 84.

Comment (SPalominos) [inline h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonWriteDriver.java]: Good question, @ebocher any idea ?

**Pull request closed without merging**